### PR TITLE
Allow setting the vpc region

### PR DESCRIPTION
**Pull request by aishwaryabk**
Signed-off-by:  Aishwarya Kamat <aishwarya.kamat@ibm.com>

### CODE REVIEW

File: openshift-install-powervs [hunk 1]
```diff
@@ -1102,6 +1102,12 @@ function variables {
 	  error "IBM Cloud VPC subnet name must be non-empty for using IBM Cloud Services"
 	fi
 	echo "use_ibm_cloud_services = \"true\"" >> $VAR_TEMPLATE
+	question "Enter IBM Cloud VPC Infrastructure region"
```
Comment (yussufsh) [inline openshift-install-powervs:1105]: Enter IBM Cloud VPC Infrastructure region. (Power VS region will be used for VPC if you do not provide any value).

Comment (aishwaryabk) [reply openshift-install-powervs]: @yussufsh updated the change.

File: docs/variables.md [hunk 2]
```diff
@@ -161,6 +161,9 @@ Targeting service crn:v1:bluemix:public:power-iaas:tor01:a/65b64c1f1c2XXXXX:4a1f
 [question] > Enter IBM Cloud VPC subnet name
 ? ocp-subnet
 - You have answered: ocp-subnet
+[question] > Enter IBM Cloud VPC Infrastructure region
```
Comment (yussufsh) [inline docs/variables.md:164]: Change here as well.

File: docs/variables.md [hunk 3]
```diff
@@ -246,5 +249,8 @@ Targeting service crn:v1:bluemix:public:power-iaas:tor01:a/65b64c1f1c2XXXXX:4a1f
...
```
Comment (yussufsh) [inline docs/variables.md:252]: Change here as well.

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: docs/variables.md (hunk 3 above)
Comment (aishwaryabk) [reply docs/variables.md]: Thanks @yussufsh

**Review by yussufsh — APPROVED** [review]
/lgtm

**Review by cs-zhang — APPROVED** [review]
/lgtm

**Review by cs-zhang — APPROVED** [review]

**Comment by cs-zhang** [comment]
@yussufsh  I have approved it, but why it does not take it?

**Comment by yussufsh** [comment]
> @yussufsh any idea? I have approved it, but why it does not take it?

Have added OWNERS file now.

**Pull request merged**